cdc/server.go

```diff
@@ -392,15 +392,30 @@ func (s *Server) setUpDataDir(ctx context.Context) error {
 		return nil
 	}
 
+	cli := s.etcdClient
```

Comment:
The new replicaImpl creates an `etcdClient` uses it when creating a new capture. The `etcdClient` won't be closed or cleaned up until the cdc server exits.

While the old replicaImpl creates a new `etcdClient` every time a server `run` is called. https://github.com/pingcap/ticdc/blob/9972085455542df023aaecb306248936a7fdcf8b/cdc/server.go#L300-L305

Maybe the new replicaImpl way is better. But I **don't think we need to** change this logic in this PR.